endpoints/API_Player.py

```diff
@@ -88,6 +88,7 @@ def add_car(self, car_details, auth_details=None):
 
     def register_car(self, car_name, brand, auth_details=None):
         "register car"
+        from conf import api_example_conf as conf
```

Comment:
Can you make `pylint` ignore `import-outside-toplevel` warning here and all other conditional imports in this PR?